src/java/org/apache/cassandra/schema/TableParams.java

```diff
@@ -105,6 +112,13 @@ private TableParams(Builder builder)
         extensions = builder.extensions;
         cdc = builder.cdc;
         readRepair = builder.readRepair;
+        automatedRepair = new HashMap<>()
```

Comment:
this breaks java8 builds, cant use <> on anonymous inner classes

Reply:
```
        automatedRepair = new EnumMap<>(AutoRepairConfig.RepairType.class);
        automatedRepair.put(AutoRepairConfig.RepairType.full, builder.automatedRepairFull);
        automatedRepair.put(AutoRepairConfig.RepairType.incremental, builder.automatedRepairIncremental);
 ```

Reply:
Incorporated